### ISSUE TITLE
Unhandled exception: Crash when updating game.

### ISSUE THREAD

**Issue by DillonSimeone** [remyroy/CDDA-Game-Launcher#75]
* Description: The last time I played was a few days ago.  I updated, and this crash resulted.
* Version: 1.3.2
* Type: `<class 'UnicodeDecodeError'>`
* Value: 'cp932' codec can't decode byte 0xef in position 0: illegal multibyte sequence
* Traceback:
```
  File "cddagl\ui.py", line 2475, in copy_next_dir
  File "cddagl\ui.py", line 2526, in post_extraction_step2
  File "cddagl\ui.py", line 2423, in asset_name

```

**Comment by DillonSimeone** [comment]
I tried again, and got the same error. Went from 0% to 100% no problem, but it appears that the crash happens after it tries to delete the old folder after updating.

**Comment by DillonSimeone** [comment]
It happens even after I run it in Adim mode. Strange.

**Comment by remyroy** [comment]
This issue has been reported multiple times. I'll check it out when I have a minute.

**Comment by remyroy** [comment]
This should be fixed with 5e038347d4af5f05c9b1fbd429f224840cc6e075 . It is now part of the new 1.3.3 version.
